tests/unit/rest.py

```diff
@@ -276,11 +276,16 @@ def one_optional(arg1, arg2=-42):
 
     # Let's also make sure we're testing something with a schema that isn't
     # just basestring:
-    @rest.rest_call(['GET', 'PUT'], '/non-string-schema',
-                    schema=Schema({"the_value": Use(int)}))
+    @rest.rest_call('PUT', '/non-string-schema',
+                    schema=Schema({"the_value": int}))
     def non_string_schema(the_value):
         return json.dumps(the_value)
 
+    @rest.rest_call('GET', '/non-string-schema2',
+                    schema=Schema({"the_value": Use(int)}))
+    def non_string_schema2(the_value):
```

Comment:
Can we come up with better variable names to distinguish these (and maybe paths to match)? maybe something that calls out the fact that one of them converts the value (with `Use`) and the other won't.